frontend/src/components/housing/submission/SubmissionIntakeForm.vue

```diff
@@ -111,14 +111,19 @@ const assistanceAssignedActivityId: Ref<string | undefined> = ref(undefined);
 const editable: Ref<boolean> = ref(true);
 const formRef: Ref<InstanceType<typeof Form> | null> = ref(null);
 const geomarkAccordionIndex: Ref<number | undefined> = ref(undefined);
+const initialFormValues: Ref<undefined | object> = ref(undefined);
+const loadForm: Ref<boolean> = ref(false);
 const mapLatitude: Ref<number | undefined> = ref(undefined);
 const mapLongitude: Ref<number | undefined> = ref(undefined);
 const mapRef: Ref<InstanceType<typeof Map> | null> = ref(null);
 const isSubmittable: Ref<boolean> = ref(false);
 const orgBookOptions: Ref<Array<any>> = ref([]);
 const parcelAccordionIndex: Ref<number | undefined> = ref(undefined);
-const validationErrors: Ref<string[]> = ref([]);
-const formModified: Ref<boolean> = ref(false);
+const validationErrors = computed(() => {
+  // Parse errors from vee-validate into a string[] of category headings
+  if (!formRef?.value?.errors) return [];
+  else return Array.from(new Set(Object.keys(formRef.value.errors).flatMap((x) => x.split('.')[0].split('[')[0])));
```

Comment:
Optional style: The else here isn't necessary as you are hard returning in the other branch on line 124.